extension/resources/payment-custom-types.json

```diff
@@ -7,6 +7,17 @@
     "payment"
   ],
   "fieldDefinitions": [
+    {
+      "type": {
+        "name": "String"
+      },
+      "name": "interfaceId",
```

Comment:
Please rename custom field `interfaceId` to `reference` or `merchantReference` as `interfaceId` has actually different meaning and purpose. It is a unique payment ID which is provided by the payment provider and not the shop itself. Adyen unfortunately does have any generalised payment specific unique IDs but rather transaction specific one only.

Reply:
This has now been changed